frontend/src/components/common/input/input.tsx

```diff
@@ -30,9 +31,28 @@ const Input: FC<Props> = ({
   type = 'text',
   hasVisuallyHiddenLabel = false,
   inputClassName = '',
+  textAreaRows = 2,
 }) => {
   const { field } = useFormControl({ name, control });
 
+  const inputArea =
+    type === 'textarea' ? (
```

Comment:
How about creating a helper function for it? (inside this component)
Something like:
```
const getInputArea = (): ReactElement {
    if (type === 'textarea') { return <textarea ... /> }
    return <input ... />
}
```

Reply:
Not sure that it is needed. Coaches what do you think?